docs/nydus-zran.md

```diff
@@ -0,0 +1,56 @@
+# Nydus zran artifact user guide
+
+This guide shows how to generate a smaller **nydus zran artifact** from an existing OCI v1 image, which is used to accelerate the image directly. It has the advantage of eliminating the need to store two images and generating much faster than converting the full accelerated image, which can save image registry storage space compared to the native nydus image. A simple test result as follows:
```

Comment:
This guide explains how to create a tiny **nydus zran artifact** from an existing OCI v1 image, which can be used to accelerate the image directly with lazy pulling. It provides several advantages because of making use of OCI blobs. First, it eliminates the need to store two separate full images, which can save massive space in your image registry. Second, generating such artifact is faster than converting the full accelerated image. A simple test result is shown below:

Reply:
Fixed, a few minor changes.